test/service/test_async_manifest_service.py

```diff
@@ -206,6 +223,10 @@ def test_manifest_endpoint_start_execution(self, mock_uuid, step_function_helper
                     self.assertEqual(301, response['Status'] if fetch else response.status_code)
                     self.assertIn('Retry-After', response if fetch else response.headers)
                     self.assertIn('Location', response if fetch else response.headers)
+                    if fetch:
+                        self.assertIn('CommandLine', response)
+                    else:
+                        self.assertNotIn('CommandLine', response.headers)
```

Comment:
This is unnecessary. 'CommandLine' isn't even valid HTTP header name. I guess your original is the best we can do here.